### ISSUE TITLE
WSL2 Can't access localhost after using flask_cors

### ISSUE THREAD

**Issue by Dogacel** [microsoft/WSL#6354]
When I run a react web server and python flask server, everything is working fine. But when I use flask_cors package and run, everything still seems fine until I make a request to the flask server. After the request is processed, I can't access to any of those addresses, localhost:5000 or localhost:3000. I can't access them until I do `wsl --shutdown` and restart. I have tried setting different ports and different hosts, eg. 0.0.0.0. I don't know what triggers it but I al so can't find WSL Adapter under Network Connections. Everything is working fine until I use flask_cors.

# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.20279.1]
Your Distribution version: Release:        20.04
Whether the issue is on WSL 2 and/or WSL 1: Linux version 4.19.128-microsoft-standard (oe-user@oe-host) (gcc version 8.2.0 (GCC)) #1 SMP Tue Jun 23 12:58:10 UTC 2020
```

# Steps to reproduce

Run a simple flask app with [flask_cors](https://flask-cors.readthedocs.io/en/latest/). Make a request. Try to connect localhost again.

**WSL logs**: 

#  Expected behavior

We should have connection to the localhost.

# Actual behavior

Localhost is inaccessible after running flask_cors web server.

**Comment by Dogacel** [comment]
It seems like this problem is solved when I add `localhostForwarding=true` to my wsl.conf. But still, why was localhost working before I have added this config until I run flask_cors? Isn't something misleading in this sceneario?

**Comment by OneBlue** [comment]
Thanks for reporting this @Dogacel.
`localhostForwarding=true` is the default, so unless you had it explicitly set to false, this shouldn't make any difference.

Can you please provide the output of `netstat -lnpt` along with the output of `curl -v http://localhost:3000` (replace with the port you're using)  inside WSL when the localhost connection fails ?

**Comment by Dogacel** [comment]
Sorry, I haven't faced this issue since then. I don't know if it is possible to access ports from localhost when localhostForwarding is false. Something sketchy happened. Maybe it was a bug when I upgraded from wsl1 to wsl2. Also command `netstat` is not found. I have removed that line from my wslconfig. Is it possible that it was defaulted to false before I have added that line? I can't reporoduce it now.
